app/BackgroundSync.ts

```diff
@@ -1,15 +1,8 @@
-import { Alert } from 'react-native';
 import RPCModule from '../components/RPCModule';
+import serverUris from './uris/serverUris';
 
-const BackgroundSync = async (task_data: Object) => {
-  Alert.alert('Alert Title', 'My Alert Msg', [
-    {
-      text: 'Cancel',
-      onPress: () => console.log('Cancel Pressed'),
-      style: 'cancel',
-    },
-    { text: 'OK', onPress: () => console.log('OK Pressed') },
-  ]);
+const BackgroundSync = async (task_data: any) => {
+  await RPCModule.initLightClient(serverUris()[0]);
```

Comment:
You have to get the server from the context (global state) because the App can have another server in settings, different than the first item in serverUris... I understand this is a testing thing.... But you have to take this in mind.

Reply:
I'm working on this, because it's not trivial to access to the context from a Background Task...